readthedocs/notifications/models.py

```diff
@@ -93,3 +93,49 @@ def get_message(self):
             )
 
         return message
+
+    def get_absolute_url(self):
```

Comment:
I don't see anything that changed here, so I assume this is just moved code. Anything in particular you want eyes on otherwise here?

Reply:
Yes, just moving the code to a place where it can be reused.